segmentstore/server/src/main/java/io/pravega/segmentstore/server/ReadIndexFactory.java

```diff
@@ -20,6 +20,7 @@
      *
      * @param containerMetadata A Container Metadata for this ReadIndex.
      * @param storage           A ReadOnlyStorage to use for reading data that is not in the cache.
+     * @return The ReadIndex object for StreamSegments
```

Comment:
I realized that all the comment lines finish with a dot, it would be great that new comments also follow this style.

Reply:
need re-submit the PR since `The sign-off is missing.` for DOC.

Reply:
#3382 opened and merged for addressing the comment. Thanks Raul for your comment.